tests/behat/features/Express/cu_users/invite.feature

```diff
@@ -1,9 +1,97 @@
-@users @invite
+@users @javascript
```

Comment:
Adding the `@javascript` tag here makes all the tests in this feature file run with the JS driver, which is really slow. I'm not sure any of these tests need the JS tag.